src/common/src/catalog/mod.rs

```diff
@@ -97,6 +97,18 @@ pub fn is_system_schema(schema_name: &str) -> bool {
     SYSTEM_SCHEMAS.iter().any(|s| *s == schema_name)
 }
 
+/// Hidden additional columns for partition/file and offset.
+/// Refer to <https://github.com/risingwavelabs/risingwave/issues/14384> for more details.
+pub const ADDITION_SPLIT_OFFSET_COLUMN_PREFIX: &str = "_rw_addition_";
+
+pub fn get_addition_key_name(col_name: &str) -> Option<&str> {
+    col_name.strip_prefix(ADDITION_SPLIT_OFFSET_COLUMN_PREFIX)
+}
+
+pub fn is_hidden_addition_col(col_name: &str) -> bool {
+    col_name.starts_with(ADDITION_SPLIT_OFFSET_COLUMN_PREFIX)
+}
```

Comment:
I am concerned about this approach. Identifying a column type by name prefix is dangerous and does not have hints on other places. 
We already have `AdditioanalColumn` as column attr, why not stick to that way?